docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,43 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create a serializable resource instance
+serializable_resource = ActiveModel::SerializableResource.new(post, options)
+
+# Serialize
+model_json = serializable_resource.as_json
+```
+
+### Retrieving a Resource's Active Model Serializer
+
+If you want to retrieve a serializer for a specific resource, you can do the following:
+
+```ruby
+# Creating the resource
+post = Post.create(title: "Another Example", body: "So much fun.")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::Serializer.serializer_for(post, options)
+```
+
+This will return an instance, if any, of the resource's serializer.
+
+You could also retrieve the serializer via: 
+
+```ruby
+ActiveModel::SerializableResource.serialize(post, options).serializer 
```

Comment:
I think we're going to soon be removing the `.serialize` method, since it turns out it is more confusing than useful.

so, like @joaomdmoura wrote `serializable_resource = ActiveModel::SerializableResource.new(post, options).serializer `

(when I wrote the method, I thought the class would be re-used for deserialization..)